spec/requests/contact_tickets_request_spec.rb

```diff
@@ -39,11 +42,12 @@
 
     it 'returns all the tickets this contact has redeemed' do
```

Comment:
maybe update the test name to say "in the correct order" to help debugging in case we ever break this?